### PR TITLE
Bittensor version updated to 6.9.4

### PR DESCRIPTION
**Pull request by the-mx**

### CODE REVIEW

File: update_validator.sh [hunk 1]
```diff
@@ -31,18 +31,18 @@ source_script "update_env.sh"
 # Navigate to the validation directory and source update script
 
 cd "../validation" || exit 1
-source_script "update_env.sh"
```
Comment (rhohndorf) [inline update_validator.sh:34]: As far as I understand, running this only rstarts the validator process. Is that what you want ?

File: neurons/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-bittensor==6.9.3
+bittensor==6.9.4
```
[the remark this answers is not in the document]
Comment (the-mx) [reply neurons/requirements.txt:1]: https://pypi.org/project/bittensor/6.9.4/

File: update_validator.sh [hunk 3]
```diff
@@ -31,18 +31,18 @@ source_script "update_env.sh"
...
+#source_script "update_env.sh"
```
[the remark this answers is not in the document]
Comment (the-mx) [reply update_validator.sh:34]: How?

File: update_validator.sh (hunk 1 above)
Comment (the-mx) [reply update_validator.sh:34]: It updates the env. Restart is performed on the last line of script

**Review by s7500 — APPROVED** [review]

**Pull request merged**